pkgs/desktops/gnome-3/devtools/gnome-builder/default.nix

```diff
@@ -0,0 +1,78 @@
+{stdenv
+, clang
+, defaultIconTheme
+, devhelp
+, fetchgit
+, gettext
+, gnome_themes_standard
+, gspell
+, gtk3
+, gtksourceview
+, hicolor_icon_theme
+, json_glib
+, jsonrpc_glib
+, libdazzle
+, libgit2-glib
+, libpeas
+, libxml2
+, llvmPackages
+, meson
+, ninja
+, pcre
+, pkgconfig
+, pkgs
+, python3
+, python3Packages
+, sysprof
+, template_glib
+, vala
+, vte
+, webkitgtk
+, wrapGAppsHook
+, ...}:
+
+stdenv.mkDerivation rec {
+  version = "3.27.2";
```

Comment:
I am aware of that. Since the 3.26 release there has been larger restructuring in build system of `gnome-builder`. I didn't want to invest time in tinkering with an older version when the (expected) release is end of February/start of march. That would also line up with NixOS 18.03.

I intend to bump the version to the next stable version as soon as those are available.

Reply:
Okay.